src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -244,7 +244,8 @@
                    formControlName="path"
                    [ngbTypeahead]="pathDataSource"
                    (selectItem)="pathChangeHandler()"
-                   (blur)="pathChangeHandler()">
+                   (blur)="pathChangeHandler()"
```

Comment:
I am not understanding why we have added this here while removing it below ?

Moreover functions should not be removed but you may just add a validation